UniWatch/EnrollViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
Why is this file in the root folder of the `UniWatch` solution? This file should be moved or deleted.